src/tools/miri/src/concurrency/vector_clock.rs

```diff
@@ -212,14 +212,16 @@ impl PartialOrd for VClock {
         for (l, r) in iter {
             match order {
                 Ordering::Equal => order = l.cmp(r),
-                Ordering::Less =>
+                Ordering::Less => {
                     if l > r {
                         return None;
-                    },
-                Ordering::Greater =>
+                    }
+                }
+                Ordering::Greater => {
                     if l < r {
                         return None;
-                    },
+                    }
+                }
```

Comment:
FWIW all the formatting changes in this file are un-done again in https://github.com/rust-lang/miri/pull/2856. It would be better to avoid formatting changes in Miri code as part of rustc PRs.

What happens here is that Miri has a diferent rustfmt.toml than rustc, so if you have format-on-save enabled then working on Miri inside the rustc tree will use incorrect formatting. I am not sure how to avoid that (IMO the rustc settings have pretty bad impact on code in Miri so I'd prefer to keep our rustfmt config).

Reply:
Yeah, sorry. I know about this problem (it often happens with clippy...), but just didn't notice this time 😓

Reply:
Sounds like a RA feature request to disable format-on-save for some subfolders...

Reply:
I think if it could use the right config it would actually be better. I'll check with r-a people if this is possible.